extensions/amp-consent/0.1/test/test-consent-state-manager.js

```diff
@@ -246,14 +246,10 @@ describes.realWin('ConsentStateManager', {amp: 1}, env => {
           instance.update(CONSENT_ITEM_STATE.ACCEPTED);
           yield macroTask();
           expect(storageSetSpy).to.be.calledOnce;
-
-          // legacy boolean consent state value
-          expect(storageSetSpy).to.be.calledWith('amp-consent:test', true);
```

Comment:
instead of removing these checks, can we please update them with value in the new format? Thanks.

Reply:
Done